### PR TITLE
Update get-software.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Odata $filter query

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5934

File: microsoft-365/security/defender-endpoint/get-software.md [hunk 1]
```diff
@@ -30,8 +30,17 @@ ms.custom: api
...
+## API description
+
 Retrieves the organization software inventory.
 
+<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
+<br>OData supported operators:
+<br>```$filter``` on:  ```id```, ```name``` and ```vendor``` properties.
+<br>```$top``` with max value of 10,000 
+<br>```$skip```
+<br>See examples at [OData queries with Microsoft Defender for Endpoint](exposed-apis-odata-samples.md)
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/get-software.md]:
```suggestion
<br>Supports [OData V4 queries](https://www.odata.org/documentation/).
<br>OData supported operators:
<br>```$filter``` on:  ```id```, ```name```, and ```vendor``` properties.
<br>```$top``` with max value of 10,000.
<br>```$skip```.
<br>See examples at [OData queries with Microsoft Defender for Endpoint](exposed-apis-odata-samples.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @msbemba, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]

**Pull request merged**